chain/epoch-manager/src/tests/mod.rs

```diff
@@ -3112,6 +3106,7 @@ fn test_max_kickout_stake_ratio() {
 }
 
 /// Common test scenario for a couple of tests exercising chunk validator kickouts.
+#[allow(deprecated)]
 fn test_chunk_validator_kickout(
     expected_kickouts: HashMap<AccountId, ValidatorKickoutReason>,
     use_endorsement_cutoff_threshold: bool,
```

Comment:
Remove `use_endorsement_cutoff_threshold` parameter and assume it to be true. Delete `test_chunk_validator_exempted`. Remove `#[allow(deprecated)]`